Source/Noesis.Javascript/JavascriptContext.h

```diff
@@ -209,15 +211,14 @@ public ref class JavascriptContext: public System::IDisposable
 	// v8 context required to be active for all v8 operations.
 	Persistent<Context>* mContext;
 
-	// Avoids us recreating these too often.
-	Persistent<ObjectTemplate> *objectWrapperTemplate;
-
 	// Stores every JavascriptExternal we create.  This saves time if the same
 	// objects are recreated frequently, and stops us building up a huge
 	// collection of JavascriptExternal objects that won't be freed until
 	// the context is destroyed.
 	System::Collections::Generic::Dictionary<System::Object ^, WrappedJavascriptExternal> ^mExternals;
 
+    System::Collections::Generic::Dictionary<System::Type ^, System::IntPtr> ^mTypeToConstructorMapping;

```

Comment:
Can we please have a comment here?  Particularly about what will be found in this IntPtr.